params/bootnodes.go

```diff
@@ -30,6 +30,10 @@ var GardenBootnodes = []string{
 	"enode://84a1545d709e862e8ee45a87558e833b8fb1ba057a093fb7c974beb51e403b2c1d6f8a89404632804cf4ba1b47e3db9fcd629feb9fa90226fd2cf2b8ce83b0c9@104.198.48.112",
 }
 
+// LocalBootnodes are the enode URLs of the P2P bootstrap nodes running on
+// the Local test network.
+var LocalBootnodes = []string{}
```

Comment:
(nit) Kinda silly to have this, since "local mode" will never have bootnodes. Thoughts on just removing this?

Reply:
yeah can do that.